packages/commonwealth/client/scripts/views/components/react_quill_editor/use_image_drop_and_paste.tsx

```diff
@@ -72,11 +69,40 @@ export const useImageDropAndPaste = ({
           app.user.jwt
         );
 
+        const selectedIndex: number = editor.getSelection()?.index;
+        const text: string = editor.getText();
+        const blankEditor: boolean = editor.getLength() === 1;
+        const isEndOfLine: boolean = text[selectedIndex].endsWith('\n');
+        var middleOfText: boolean = false;
+
+        // adds line break to insert image above or below text
+        if (!blankEditor) {
+          var lineBreak: string = '\n';
```

Comment:
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/9f208b3e-e4f3-4d7b-b847-d0498063fc25)

make sure you do not use `var`s